### ISSUE TITLE
[a11y extension] Foreign maction interference

### ISSUE THREAD

**Issue by zorkow** [mathjax/MathJax-RespEq#102]
Mathml already containing mactions can lead to strange behaviour and potentially incorrect speech output.

**Comment by pkra** [comment]
Was this fixed?

**Comment by zorkow** [comment]
No, it was not properly addressed. There is currently no treatment of mactions in the semantic enrichment. They are effectively being ignored.

**Comment by zorkow** [comment]
No, it was not properly addressed. There is currently no treatment of mactions in the semantic enrichment. They are effectively being ignored.
